ui/components/src/TimeChart/TimeChart.tsx

```diff
@@ -270,10 +267,28 @@ export const TimeChart = forwardRef<ChartInstance, TimeChartProps>(function Time
     __experimentalEChartsOptionsOverride,
     noDataVariant,
     timeZone,
-    tooltipPinnedCoords,
     isStackedBar,
+    enablePinning,
+    pinnedCrosshair,
   ]);
 
+  // Update adjacent charts so tooltip is unpinned when current chart is clicked.
+  useEffect(() => {
+    // Only allow pinning one tooltip at a time, subsequent tooltip click unpins previous.
+    // Multiple tooltips can only be pinned if Ctrl or Cmd key is pressed while clicking.
+    if (tooltipPinnedCoords !== null && lastTooltipPinnedCoords !== null) {
+      if (!isEqual(lastTooltipPinnedCoords, tooltipPinnedCoords)) {
+        setTooltipPinnedCoords(null);
+      }
+      const isCrosshairPinned = checkCrosshairPinnedStatus(seriesMapping);
+      if (tooltipPinnedCoords !== null && isCrosshairPinned) {
+        setPinnedCrosshair(null);
+      }
+    }
+    // tooltipPinnedCoords CANNOT be in dep array or tooltip pinning breaks in the current chart's onClick
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [lastTooltipPinnedCoords, seriesMapping]);
```

Comment:
Should this be `updatedSeriesMapping`?

Reply:
updatedSeriesMapping is only inside the useMemo currently, going to need to further refactor this approach further now so converting back to Draft PR